src/App.css

```diff
@@ -1,39 +1,60 @@
+body {
```

Comment:
- [ ] All the boilerplate that got created by the CRA (texts, images, and styles) should be deleted, lie the images and logos on public folder